### PR TITLE
noa shoval author info

### PR DESCRIPTION
**Pull request by noashoval**
### Description
Adds noashoval as an OSCI author to the repository
 
### Issues Resolved
N/A

### Check List
- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _authors/noashoval.markdown [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+name: Noa Shoval
+short_name:Noa
```
Comment (cwperks) [inline _authors/noashoval.markdown:3]: nit: Please add a space between `short_name:` and your name.

**Review by noashoval — COMMENTED** [review]
Added space between short_name: and name

File: _authors/noashoval.markdown (hunk 1 above)
Comment (noashoval) [reply _authors/noashoval.markdown]: added the space!

**Review by dtaivpp — CHANGES_REQUESTED** [review]
Hey actually one more change before we can merge, can you move your bio from the `_authors` to the `_authors/osci_2023`

**Comment by dtaivpp** [comment]
Closing as we are past deadline and there are outstanding changes.

**Pull request closed without merging**